jobs/scripts/dependencies-ztc.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
-pip install -r requirements.txt
-buildout buildout:git-clone-depth=1 -c core.cfg
+pip install -Ur requirements.txt
+buildout buildout:git-clone-depth=1
```

Comment:
Why removing the ``-c core.cfg``? ``buildout.cfg`` is only adding ``Plone`` egg and some sphinx thingies, we don't need those on CI IMHO